packages/visx-brush/src/types.ts

```diff
@@ -44,6 +44,12 @@ export type ResizeTriggerAreas =
   | 'bottomLeft'
   | 'bottomRight';
 
+export type BrushingType = 'move' | 'select' | ResizeTriggerAreas;
+export type BrushingOptions = {
```

Comment:
`BrushingOptions` is pretty generic and so is a little hard to understand what it means. Could we update to something like `BrushPageOffset`?